source/docs/software/wpilib-tools/axon/labeling-a-dataset-with-supervisely.rst

```diff
@@ -0,0 +1,62 @@
+Labeling a Dataset with Supervisely
+===================================
+
+The quality of a neural network is fully dependent on the quality of the dataset used to train it. To create a custom dataset, perhaps to train a neural network to recognize the new game piece, Axon supports datasets created using `Supervisely <https://supervise.ly>`__. This page describes how to create label, and export a dataset made with Supervisely.
+
+Record a Video
+--------------
+
+The first step to creating a dataset is getting images. A dataset should have several thousand images. To get this many images, the easiest way is to record a short video, roughly 1 to 5 minutes long. The video should clearly display the object(s) you wish to detect in several angles. Below is an example clip of a video used to teach a neural network to recognize a video game controller.
+
+
+.. raw:: html
+
+    <iframe width="470" height="355" src="https://www.youtube.com/embed/7fPkAV7a0ZM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope" allowfullscreen></iframe>
```

Comment:
```suggestion
    <iframe width="470" height="355" src="https://www.youtube.com/embed/7fPkAV7a0ZM" title="YouTube video player" frameborder="0" allow="autoplay; encrypted-media;" allowfullscreen></iframe>
```

Reply:
and should you replace this with the wpi hosted link?

Reply:
yep, it was a temporary solution. Next commit.